recipes/intx/all/conanfile.py

```diff
@@ -0,0 +1,75 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import get, copy, export_conandata_patches, apply_conandata_patches
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.layout import basic_layout
+import os
+
+required_conan_version = ">=1.51.1"
+
+class IntxConan(ConanFile):
+    name = "intx"
+    description = "Extended precision integer C++ library"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/chfast/intx"
+    topics = ("evm", "biginteger", "arbitrary-precision", "header-only")
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
+
+    @property
+    def _min_cppstd(self):
+        return 20
+
+    def export_sources(self):
+        export_conandata_patches(self)
```

Comment:
Hi @pwqbot, thanks for the contribution. This should not be needed if there are not patches.
```suggestion
```